web/src/screens/InterviewRecordingPage.tsx

```diff
@@ -58,13 +59,14 @@ const InterviewRecordingPage = () => {
       );
       if (response.data.status === 'success') {
         // caseSensitiveAxiosInstance doesn't work
-        const interview = response.data.payload.interview;
-        interview.quizzes.map((quiz: any) => {
+        const theInterview = response.data.payload.interview;
```

Comment:
Would interviewPayload be more descriptive?

Reply:
That's a better name for sure 👍